src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -66,7 +66,7 @@ public static void setDatesToRepoConfigs(
             List<RepoConfiguration> configs, Optional<Date> sinceDate, Optional<Date> untilDate) {
         for (RepoConfiguration config : configs) {
             config.setSinceDate(sinceDate.orElse(null));
-            config.setUntilDate(untilDate.orElse(null));
+            config.setUntilDate(untilDate.orElse(new Date()));
```

Comment:
Can add a comment above this, that the `new Date()` is for setting the `untilDate` to be the current/report generation date?